frontend/integration-tests/views/secrets.view.ts

```diff
@@ -59,11 +60,12 @@ export const checkSecret = async(ns: string, name: string, keyValuesToCheck: Obj
   expect(renderedKeyValues).toEqual(keyValuesToCheck);
 };
 
-export const editSecret = async(ns: string, name: string, updateForm: Function) => {
+export const editSecret = async(ns: string, name: string, elementToWaitFor: ElementFinder, updateForm: Function) => {
```

Comment:
It would be nice if this didn't need to be passed in. Is there a common element on all secret forms we could wait for?

Reply:
yeah, the secret name input field